addons/mail/static/src/messaging/entity/messaging_initializer/messaging_initializer.js

```diff
@@ -39,7 +39,7 @@ function MessagingInitializerFactory({ Entity }) {
                 name: this.env._t("History"),
             });
 
-            this.messaging.update({
+            this.env.messaging.update({
```

Comment:
Why the change in this file?

Reply:
because `messaging` is no longer a field on `Entity`, and no inverse is defined on `Messaging.fields.initializer`. Should we introduce it back?

Reply:
messaging on Entity, no.
inverse field, maybe.
But there is still the problem that it is a one2one that should be "required" on both sides in this case, so one side will always have the problem that it is "not set immediately".

Reply:
I think we can delay all the comments related to this until at least we have required fields, and probably even until "mixin" stuff as you mentioned somewhere.

As fallback `env.messaging` can always be used in the mean time.

Reply:
no immediately if you consider direct updates phase of an update cycle, but since this is an explicit update it will work fine for any computed fields

Reply:
I mean, if the field becomes "required" on both side, it should be set at "create" on both sides, but both cannot be created at the same time.
This is a clear example that it should be the same entity but we just split it in two for smaller files.
So only mixin can solve this, and/or initializer and handlers should just "patch" the original messaging with the extra features for the sake of code separation.

Reply:
Maybe autocreate could allow to bypass the check of required fields until the creating of both has taken place.